### PR TITLE
Docs: Clean up of 'blockinfile' module docs

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This is part of a series of module doc cleanups.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
blockinfile

##### ANSIBLE VERSION
v2.8

### CODE REVIEW

File: lib/ansible/modules/files/blockinfile.py [hunk 1]
```diff
@@ -12,102 +12,95 @@
...
+      get the original file back if you somehow clobbered it incorrectly.
     type: bool
-    default: 'no'
+    default: no
   marker_begin:
     description:
-      - This will be inserted at {mark} in the opening ansible block marker.
-    default: 'BEGIN'
-    version_added: "2.5"
+    - This will be inserted at {mark} in the opening ansible block marker.
```
Comment (acozine) [inline lib/ansible/modules/files/blockinfile.py]: should {mark} (here and line 83) be in C( . . . )?

**Comment by acozine** [comment]
Thanks @dagwieers everything else looks good.

File: lib/ansible/modules/files/blockinfile.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/files/blockinfile.py]: Yes it should, these are values or pieces of values. Thanks !

**Review by acozine — COMMENTED** [review]
Nice cleanup, and thanks for splitting these changes into a few PRs. It makes it easier for me to track which sections still need review. Let me know what you think about the {mark} references. Otherwise this one's ready to merge.

File: lib/ansible/modules/files/blockinfile.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/files/blockinfile.py]: Fixed.

**Comment by dagwieers** [comment]
rebuild_merge

**Pull request merged**